Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.9-slim
```

Comment:
sem neni potreba davat i nejaky OS? Jakoze treba ubuntu image s pythonem?

Reply:
jo a na docker veci jsem tam prave nachystal tu docker slozku. Ten Dockerfile jsem zamyslel prave tam.

Reply:
slim je oklestena verze ubuntu, spravne by tam mela byt i konkretni verze, ale asi jsem byl liny

Reply:
> jo a na docker veci jsem tam prave nachystal tu docker slozku. Ten Dockerfile jsem zamyslel prave tam.

ja jsem to pochopil, ale kdyz neni Dockerfile v root slozce, tak je s tim vic prace :sweat_smile:

Reply:
> jo a na docker veci jsem tam prave nachystal tu docker slozku. Ten Dockerfile jsem zamyslel prave tam
@jakub-h fixed

Reply:
pravda, ze my to v praci delame tak, ze do te docker slozky pridavame i `build.sh` a `run.sh` soubory s nejakyma nasema specialitkama, takze je mozny, ze tim obcahzime tu slozitost non-root umisteni Dockerfilu :smile: